src/dash-table/dash/DataTable.js

```diff
@@ -423,6 +427,15 @@ export const propTypes = {
         separate_4digits: PropTypes.bool
     }),
 
+    markdown_options: PropTypes.exact({
+        linkTarget: PropTypes.oneOf([
+            '_blank',
+            '_parent',
+            '_self',
+            '_top'
```

Comment:
Can we just make this a string, to support named windows/frames too, and document the special values in the prop description?

Reply:
Sure. Even though it's redundant, I think I'll define the type as :
```
PropTypes.oneOfType([
            Proptypes.string,
            PropTypes.oneOf([
                '_blank',
                '_parent',
                '_self',
                '_top'
            ])
        ])
``` 
so the special values are documented eplicitly as part of the API.